docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+.. post:: October 9, 2017
+   :tags: newsletter
+
+
+#####################################
+Write the Docs Newsletter – October
+#####################################
+
+Hello friends and fellow documentarians!
+
+Kelly O. here, happy to be arriving in your inbox for another round-up of docs-wisdom gleaned from our community. It's been a big month, since last we spoke. We had our fourth European conference, we've surpassed 2500 members in Slack, AND it's officially been a year since we started this here newsletter experiment. Happy newsletterversary!
```

Comment:
Remove comma between "month" and "since"

Reply:
Also, love "newsletterversary!"

Reply:
done and thanks! ;)